api/v1beta1/consts.go

```diff
@@ -142,3 +142,18 @@ const (
 	// UpdatingReason means the resource is being updated.
 	UpdatingReason = "Updating"
 )
+
+const (
+	// CustomHeaderPrefix is the prefix of annotations that enable additional cluster / node pool features.
+	// Whatever follows the prefix will be passed as a header to cluster/node pool creation/update requests.
+	// E.g. add `"infrastructure.cluster.x-k8s.io/custom-header-UseGPUDedicatedVHD": "true"` annotation to
+	// AzureManagedMachinePool CR to enable creating GPU nodes by the node pool.
+	CustomHeaderPrefix = "infrastructure.cluster.x-k8s.io/custom-header-"
```

Comment:
same here https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/azure/const.go

Reply:
In fact in this case it looks like `CustomHeaderPrefix` under the `azure` package is not used.